azure-pipelines.yml

```diff
@@ -1,54 +1,97 @@
-trigger:
```

Comment:
Be careful with the file encoding. There's an issue where UTF8 with BOM isn't being correctly handled. This file needs to be either ANSI or UTF8 without a BOM.